scripts/tasks/ts.ts

```diff
@@ -66,15 +66,6 @@ export const ts = {
       ...(useTsBuildInfo && { tsBuildInfoFile: '.amd.tsbuildinfo' }),
     });
 
-    return tscTask(options);
-  },
-  commonjsOnly: () => {
```

Comment:
this was never used ?

Reply:
it was used for node packages only - I refactored that in this PR

Reply:
also thanks to failing pipelines , it was used in non v9 packages to trigger or not api-extractor runs ...

https://github.com/microsoft/fluentui/commit/fd41e5b15e3a2f08245cd4e4bbcc0f07a367ce3b#diff-ce580dd7bb5c4a3e100928eb7b9a140f97e504eeffdb0e7f90e1983f1e3f871d